web/profiles/joinup/joinup.post_update.php

```diff
@@ -11,3 +11,10 @@
 function joinup_post_update_install_views_data_export() {
   \Drupal::service('module_installer')->install(['views_data_export']);
 }
+
+/**
+ * Enable the "OpenEuropa Newsroom Newsletter" module.
+ */
+function joinup_post_update_install_oe_newsroom_newsletter() {
```

Comment:
I was originally planning to put the code there, but then I saw that `joinup_newsletter` just contains the configuration of the "newsletter" node type that is used to store the old newsletters that were migrated from the D6 site. It doesn't offer any functionality of subscribing to newsletters.

I figured that this was just like a typical content type "feature module" and is probably destined to be removed in the future if we decide to get rid of the backed up old newsletters. I wouldn't put any new code in here right now since it will just make it more complicated to remove in the future.

Since most of the functionality is offered by the contrib module I thought it was overkill to put the post update hook in a separate module.

Reply:
@pfrenssen but you already did. Check your PR :)

Reply:
Actually I decided that I don't care so much about where this code is located, and it appears you care at least enough to make this comment, so I moved it into `joinup_newsletter` after all :stuck_out_tongue_closed_eyes: